### PR TITLE
adding Rspec, FactoryGirl, Guard, TravisCI

### PR DESCRIPTION
**Pull request by peterclark**

### CODE REVIEW

File: app/controllers/accounts_controller.rb [hunk 1]
```diff
@@ -1,9 +1,7 @@
 class AccountsController < ApplicationController
-  before_action :authenticate_user!
```
Comment (eanders) [inline app/controllers/accounts_controller.rb]: @peterclark this looks like something we shouldn't do.  We require authentication everywhere unless explicitly skipped for public pages/controllers.  Can we make travis/rspec work with this in place?

File: spec/controllers/accounts_controller_spec.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
```
Comment (eanders) [inline spec/controllers/accounts_controller_spec.rb]: Any reason to include this file if it's not implemented yet?

File: app/controllers/accounts_controller.rb (hunk 1 above)
Comment (peterclark) [inline app/controllers/accounts_controller.rb]: ':authenticate_user' is inherited from the 'application_controller.rb' file.

Comment (eanders) [inline app/controllers/accounts_controller.rb]: Ah, sorry, must be late, I mis-read the file name.  Thanks for the clarification

File: spec/controllers/accounts_controller_spec.rb (hunk 2 above)
Comment (peterclark) [inline spec/controllers/accounts_controller_spec.rb]: just a placeholder for a controller test I need to get working - I can uncomment the code and ad `skip` so it shows up in the tests output.

**Pull request merged**